hazelcast/src/test/java/com/hazelcast/client/connectionstrategy/ConfiguredBehaviourTest.java

```diff
@@ -113,6 +114,19 @@ public void testAsyncStartTrue() {
         });
     }
 
+    @Test
+    public void testAsyncStartTrueShouldNotBlock_whenThereIsClientStateToSend() {
+        class A { }
+        ClientConfig clientConfig = new ClientConfig();
+        clientConfig.getConnectionStrategyConfig().getConnectionRetryConfig().setClusterConnectTimeoutMillis(-1);
```

Comment:
Isn't this the default? Why set it again?

Reply:
To be explicit if the default changes, as I remember.